module/platform/NUSense/HardwareIO/src/NUSenseParser.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef MODULE_PLATFORM_NUSENSE_HARDWAREIO_NUSENSE_PARSER_HPP
+#define MODULE_PLATFORM_NUSENSE_HARDWAREIO_NUSENSE_PARSER_HPP
+
+#include <array>
+#include <cstdint>
+#include <memory>
+#include <vector>
+
+namespace module::platform::NUSense {
+
+    struct NUSenseFrame {
+        std::array<uint8_t, 3> header{};  //< NBS Header, should be 0xE2, 0x98, 0xA2
```

Comment:
Same for the other fields
```suggestion
        /// NBS Header, should be 0xE2, 0x98, 0xA2
        std::array<uint8_t, 3> header{};
```

Reply:
///< is a totally viable alternate doxygen syntax when you want to be more concise!

Reply:
Consistency trumps "valid alternative style"

Reply:
I kinda like it for short things

Reply:
I chose consistency